docs/examples/grpc/README.md

```diff
@@ -1,91 +1,166 @@
 # gRPC
 
-This example demonstrates how to route traffic to a gRPC service through the
-nginx controller.
+This example demonstrates how to route traffic to a gRPC service through the nginx controller.
 
 ## Prerequisites
 
 1. You have a kubernetes cluster running.
-2. You have a domain name such as `example.com` that is configured to route
-   traffic to the ingress controller.  Replace references to
-   `fortune-teller.stack.build` (the domain name used in this example) to your
-   own domain name (you're also responsible for provisioning an SSL certificate
-   for the ingress).
-3. You have the nginx-ingress controller installed in typical fashion (must be
-   at least
-   [quay.io/kubernetes-ingress-controller/nginx-ingress-controller:0.30.0](https://quay.io/kubernetes-ingress-controller/nginx-ingress-controller)
-   for grpc support.
-4. You have a backend application running a gRPC server and listening for TCP
-   traffic.  If you prefer, you can use the
-   [fortune-teller](https://github.com/kubernetes/ingress-nginx/tree/master/images/grpc-fortune-teller)
-   application provided here as an example.
-
-### Step 1: kubernetes `Deployment`
+2. You have a domain name such as `example.com` that is configured to route traffic to the ingress controller. You're also responsible for provisioning an SSL certificate for the ingress).
+3. You have the nginx-ingress controller installed as per docs for gRPC support.
+4. You have a backend application running a gRPC server and listening for TCP traffic.  If you want, you can use https://raw.githubusercontent.com/grpc/grpc-go/master/examples/features/reflection/server/main.go as an example.
 
+### Step 1: Create kubernetes `Deployment` of gRPC app
```

Comment:
Let's fix old mistakes: Kubernetes (upper K) :)

Reply:
Changed. Will appear in next push